include/zephyr/dt-bindings/interrupt-controller/tcc-tic.h

```diff
@@ -0,0 +1,158 @@
+#define TIC_EXT3                  (TIC_SPI_START + 74UL)
+#define TIC_EXT4                  (TIC_SPI_START + 75UL)
+#define TIC_EXT5                  (TIC_SPI_START + 76UL)
+#define TIC_EXT6                  (TIC_SPI_START + 77UL)
+#define TIC_EXT7                  (TIC_SPI_START + 78UL)
+#define TIC_EXT8                  (TIC_SPI_START + 79UL)
+#define TIC_EXT9                  (TIC_SPI_START + 80UL)
+#define TIC_EXTn0                 (TIC_SPI_START + 81UL)
+#define TIC_EXTn1                 (TIC_SPI_START + 82UL)
+#define TIC_EXTn2                 (TIC_SPI_START + 83UL)
+#define TIC_EXTn3                 (TIC_SPI_START + 84UL)
+#define TIC_EXTn4                 (TIC_SPI_START + 85UL)
+#define TIC_EXTn5                 (TIC_SPI_START + 86UL)
+#define TIC_EXTn6                 (TIC_SPI_START + 87UL)
+#define TIC_EXTn7                 (TIC_SPI_START + 88UL)
+#define TIC_EXTn8                 (TIC_SPI_START + 89UL)
+#define TIC_EXTn9                 (TIC_SPI_START + 90UL)
+#define TIC_FMU_IRQ               (TIC_SPI_START + 91UL)
+#define TIC_FMU_FIQ               (TIC_SPI_START + 92UL)
+#define TIC_I2S_DMA               (TIC_SPI_START + 93UL)
+#define TIC_GMAC                  (TIC_SPI_START + 94UL)
+#define TIC_GMAC_TX0              (TIC_SPI_START + 95UL)
+#define TIC_GMAC_TX1              (TIC_SPI_START + 96UL)
+#define TIC_GMAC_RX0              (TIC_SPI_START + 97UL)
+#define TIC_GMAC_RX1              (TIC_SPI_START + 98UL)
+#define TIC_SRAM_ECC              (TIC_SPI_START + 99UL)
+#define TIC_DMAC_                 (TIC_SPI_START + 100UL)
+#define TIC_DMAC_INT_CLEAR        (TIC_SPI_START + 101UL)
+#define TIC_DMAC_INTERR           (TIC_SPI_START + 102UL)
+#define TIC_HSM_WDT_OUTPUT        (TIC_SPI_START + 103UL)
+#define TIC_MBOX_SLV_TX           (TIC_SPI_START + 104UL)
+#define TIC_MBOX_MST_TX           (TIC_SPI_START + 105UL)
+#define TIC_AES                   (TIC_SPI_START + 106UL)
+#define TIC_TRNG                  (TIC_SPI_START + 107UL)
+#define TIC_TRNG_FAD              (TIC_SPI_START + 108UL)
+#define TIC_PKE                   (TIC_SPI_START + 109UL)
+#define TIC_PKE_ECC_S             (TIC_SPI_START + 110UL)
+#define TIC_PKE_ECC_D             (TIC_SPI_START + 111UL)
+#define TIC_HSM_DEFAULT_SLV_ERR   (TIC_SPI_START + 112UL)
+#define TIC_HSM_CPU_SYS_RESET_REQ (TIC_SPI_START + 113UL)
+#define TIC_PFLASH                (TIC_SPI_START + 114UL)
+#define TIC_DFLASH                (TIC_SPI_START + 115UL)
+#define TIC_RTC_ALARM             (TIC_SPI_START + 116UL)
+#define TIC_RTC_WAKEUP            (TIC_SPI_START + 117UL)
+#define TIC_GMAC_TX2              (TIC_SPI_START + 118UL)
+#define TIC_GMAC_RX2              (TIC_SPI_START + 119UL)
+#define TIC_SFMC1                 (TIC_SPI_START + 120UL)
+#define TIC_SPU                   (TIC_SPI_START + 121UL)
+
+#define TIC_EINT_START_INT (TIC_EXT0)
+#define TIC_EINT_END_INT   (TIC_EXT9)
+#define TIC_EINT_NUM       ((unsigned long)TIC_EXTn0 - (unsigned long)TIC_EXT0)
```

Comment:
```suggestion
#define TIC_EINT_NUM       ((unsigned long)TIC_EXT_N0 - (unsigned long)TIC_EXT0)
```
(Please forgive me, but I don't understand why the conversion is necessary. It seems redundant.)